tests/unit/Search/SearchUIUtilsTest.ts

```diff
@@ -2762,7 +2762,7 @@ describe('SearchUIUtils', () => {
 
             SearchUIUtils.createAndOpenSearchTransactionThread(transactionListItem, iouReportAction, hash, backTo);
 
-            expect(createTransactionThreadReport).toHaveBeenCalledWith(report1, iouReportAction);
+            expect(createTransactionThreadReport).toHaveBeenCalledWith(report1, iouReportAction, transactionID);
```

Comment:
Can you please add a full test to cover this specific flow where the IOU report action is missing?